### DISCUSSION TITLE
cant get it out of setup mode

### DISCUSSION THREAD

**Discussion by jaygrovr** [jomjol/AI-on-the-edge-device#2485] [Q&A] [answered]
Hi guys, this work is amazing by the way.  I cant wait to get it going with several meters I have in my home.  I started with my water softener which only has digital digits on the amount of gallons used.

I flashed both the esp32 and the micro SD, setup seems to happen and I get wifi connection to the device.  I went through the steps, and turned off analog meters, since I didnt have any.  After I get to the last step and reboot and switch from setup mode to normal mode, the unit still resets and comes back into setup mode.

Any idea why this is happening?
Also I wanted to redo all my ROIs and I deleted them all from digital, now I cannot create any new ones.  When I press "New ROI (after current)", nothing happens.

<img width="623" alt="image" src="https://github.com/jomjol/AI-on-the-edge-device/assets/7143066/759ba729-84ca-43be-ace8-c4c534df1b67">

**Comment by friedpa** [comment, marked as answer]
Look in the config,ini if 

SetupMode = false

**Comment by jaygrovr** [comment]
if it keeps booting into setup mode, how can I ever get to the file manager to change the config file?

**Reply by Slider0007** [reply]
You have to force refresh your screen (Ctrl+F5) or clear your browser cache.

**Reply by jaygrovr** [reply]
ok that worked! thank you.

Now having issues with the OCR part.. I feel like I have clear numbers, but its not reading them correctly.  Although it says its still aligning, I set the rotation correctly, but the image in the overview window looks rotated.

<img width="737" alt="image" src="https://github.com/jomjol/AI-on-the-edge-device/assets/7143066/02f3c85e-d6dc-4540-874f-1fb566b8f38f">

Also now having issues with the alignment marks.. i cannot save a new one since it says its always busy.  I added some new registration marks, but its not letting me "update reference image".

**Reply by Slider0007** [reply]
> Also now having issues with the alignment marks.. i cannot save a new one since it says its always busy. I added some new registration marks, but its not letting me \"update reference image\".

Make sure your round interval is not too short (e.g. 5min) and wait until state 'flow finished' and then update the alignment marker.

**Reply by jaygrovr** [reply]
Ok its all working for me now, thanks so much.  What is the real update time for this system?  How often does it take a pic and analyze the image?  I dont understand the rounding function?  My display is pure digital, so the number will always be static, no partials.  After I changed the type of digit class to 11 it started working much better.
Are there any plans to add LCD graphic detection?  Basically if a symbol is shown or not, to make a binary output?
